README.md

```diff
@@ -139,6 +139,96 @@ command groups. The `NullOrchestrator` disables commands related to managing ser
 orchestrator=NullOrchestrator()
 ```
 
+#### HelmOrchestrator
+
+The project also includes a [helm](https://helm.sh/docs/intro/quickstart/) orchestrator for deploying charts to [kubernetes](https://kubernetes.io/) clusters.
+
+Create a new `resources` directory as follows:
+
+```bash
+drwxrwxr-x  resources/
+.rw-rw-r-- ├──  settings.yml
+drwxrwxr-x └──  templates/
+drwxrwxr-x    ├──  baseline/
+drwxrwxr-x    │  └──  cli/
+drwxrwxr-x    │     └──  helm/
+drwxrwxr-x    │        ├──  values/
+.rw-rw-r--    │        │  └──   values.yml
+drwxrwxr-x    │        ├──  values-files/
+.rw-rw-r--    │        └──   mychart.tgz
+drwxrwxr-x    └──  configurable/
+drwxrwxr-x       └──  cli/
+drwxrwxr-x          └──  home/
+drwxrwxr-x             └──  .kube/
+.rw-------                └──  config  # Overwrite this with a cluster specific config file. ie `~/.kube/config`.
+```
+
+You can then define the orchestrator:
+
+```python
+from appcli.orchestrators import HelmOrchestrator
+orchestrator = HelmOrchestrator(
+    # Name of the project.
+    release_name="myapp",
+
+    # Chart archive path (relative to `conf/templates/cli/helm/`).
+    # [Optional] Default is `chart/`
+    chart_location="mychart.tgz",
+
+    # Chart source path (absolute).
+    # [Optional] Default is `None`.
+    dev_chart_location="path/to/mychart"
+)
+```
+
+_NOTE:_ Using `dev_chart_location` allows you to specify the location to your charts source code, so that live changes can be reflected every time `./myapp service start` is run.
+The following conditions must be met in order to use it:
+
+1. `dev_chart_location` is set, and is an **ABSOLUTE** path to the chart directory.
+2. Your application must be running in `dev_mode` as validated by:
+
+```python
+appcli.cli_context.CliContext.is_dev_mode
+```
+
+##### Values
+
+Values can be supplied either:
+
+1. Globally for any files dumped in the `values` directory.
+2. For a set key by dumping files in `values-files` at that nested key layer.
+
+Take the following `cli/helm/` directory structure:
+
+```bash
+drwxrwxr-x  cli/helm/
+drwxrwxr-x ├──  values/
+.rw-rw-r-- │  └──  foobar.yml
```

Comment:
Include another file in here, so it's obvious that they can have multiple files.